Browser_IDE/stylesheet.css

```diff
@@ -303,11 +348,34 @@ li.CodeMirror-hint-active {
     color: white;
 }
 
+#fileViewContainer,
+#codeViewContainer,
+#runtimeContainer {
+  padding: 10px;
```

Comment:
This is just personal opinion, but I don't really like how this looks, as it breaks up the lines in the interface and makes things bulkier. You can keep it if you think it really looks better, but I'd prefer it removed I guess.